test/commands/extended/get_bundles_test.py

```diff
@@ -308,15 +333,45 @@ def test_happy_path(self):
             'trytes': [self.spam_trytes],
         })
 
-        response = self.command(transaction = self.tx_hash)
+        response = self.command(transactions = [self.tx_hash])
+
+        self.maxDiff = None
```

Comment:
💭 I see that we do this in a few places (and I am including me in that "we" 😅); it just occurred to me that this change doesn't get reverted after each test.  Is that something we should fix?

Reply:
What behavior do we exactly want? ``maxDiff=None`` disables the limit on diff output, does it affect the actual test result, or does it only makes the output visible when it fails? Could you elaborate a bit more on what should be fixed? :smiley:

Reply:
Yeah, that's true.  I guess it's just the purist in me — modifying a variable outside the test scope and not restoring the value after the test finishes? 🙀 Blaphemy! 😅

Okiedoke; no need to change this 😸